### DISCUSSION TITLE
New update shows 0 RPM under 20-30% on CPU fan

### DISCUSSION THREAD

**Discussion by conrad22222** [Rem0o/FanControl.Releases#2559] [Q&A]
Not sure what changed because my CPU fan used to seemingly show a smooth curve all the way down to 0% and my case fans still do. It doesn't really change much since I usually have minimum fan speed at 40% but curious if it would still work if they were set at 20% as it shows 0RPM there now.

**Comment by Rem0o** [comment]
What's the start/stop% set on it?

**Comment by conrad22222** [comment]
It hasn't mattered because I normally use a 40% minimum but when
calibrating or if I try to manually set it lower than like 28% is shows as
0 rpm

**Reply by ionGL** [reply]
Just try and set Start, Stop and Minimum % to 0 and then see how manually setting 28% works.

**Reply by Rem0o** [reply]
By the way, no fan has a smooth curve to 0 RPM, they all have a cutoff point.
